homework-2/src/app/components/feature/reviews/ReviewList.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Box } from "@chakra-ui/react";
+import { IReviewItemProps, ReviewItem } from "./ReviewItem";
+import { IReview } from "../shows/ShowReviewSection ";
+
+interface IReviewListProps {
+  reviews: {
```

Comment:
This type is an array of reviews. You should extract the review typings to `interface IReview` and use it here and in other places.